src/pages/learn/introspection.mdx

```diff
@@ -1,23 +1,41 @@
 # Introspection
 
-It's often useful to ask a GraphQL schema for information about what
-queries it supports. GraphQL allows us to do so using the introspection
-system!
+<p className="learn-subtitle">Learn how to query information about a GraphQL schema</p>
 
-For our Star Wars example, the file
-[starWarsIntrospection-test.ts](https://github.com/graphql/graphql-js/blob/main/src/__tests__/starWarsIntrospection-test.ts)
-contains a number of queries demonstrating the introspection system, and is a
-test file that can be run to exercise the reference implementation's
-introspection system.
+It's often useful to ask a GraphQL schema for information about what operations it supports. GraphQL allows us to do so using the [introspection system](https://spec.graphql.org/draft/#sec-Introspection).
```

Comment:
Perhaps "features" is safer, avoids confusion with operation types (query/mutation/subscription), but either way it's better than my arch-nemesis "queries" :grin: 

```suggestion
It's often useful to ask a GraphQL schema for information about what features it supports. GraphQL allows us to do so using the [introspection system](https://spec.graphql.org/draft/#sec-Introspection).
```